src/pycatcher/diagnostics.py

```diff
@@ -365,7 +365,7 @@ def build_stl_outliers_plot(df) -> plt:
         plot: A plot with detected outliers.
     """
 
-    # logging.info("Starting outlier detection using STL")
+    logging.info("Starting outlier detection using STL")
```

Comment:
Re-enabled the logging. Please let me know if you think we don't need this on purpose.